_sources/SimplePythonData/StatementsandExpressions.rst

```diff
@@ -0,0 +1,141 @@
+..  Copyright (C)  Brad Miller, David Ranum, Jeffrey Elkner, Peter Wentworth, Allen B. Downey, Chris
+    Meyers, and Dario Mitchell.  Permission is granted to copy, distribute
+    and/or modify this document under the terms of the GNU Free Documentation
+    License, Version 1.3 or any later version published by the Free Software
+    Foundation; with Invariant Sections being Forward, Prefaces, and
+    Contributor List, no Front-Cover Texts, and no Back-Cover Texts.  A copy of
+    the license is included in the section entitled "GNU Free Documentation
+    License".
+
+.. qnum::
+   :prefix: data-9-
+   :start: 1
+
+Statements and Expressions
```

Comment:
I think there are some good visualizations we can pull out of thinkcspy section that illustrate precedence as well as how statements and expressions are evaluated by Python.  I'd be in favor of pulling those into this section.

Reply:
@laumurph: Is this version already meant to merge/include stuff from thinkcspy? Had you already noticed those visualizations and decided not to include them? Or is @bnmnetp referring to the showeval directive, which we use in the OrderOfOperations page instead of here?

Reply:
This version is meant to include stuff from thinkcspy, but I tended to favor the content from pip over thinkcspy since I thought we wanted the content to look more like pip and have the underlying structure look more like thinkcspy. @bnmnetp for the visualization, were you referring to the example that uses the python shell? I had left that out since the pip textbook tried to steer clear of using python shell examples early on. I could add that back in as looks now in thinkcspy or I could convert it into a showeval example if that is the visualization you're referring to.